hack/install-vm.sh

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env bash
+​
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+​
+INSTALL_PATH="/usr/sbin"
+ANTREA_AGENT="antrea-agent"
+AGENT_LOG_DIR="/var/log/antrea"
+AGENT_CONF_PATH="/etc/antrea"
+OVS_BRIDGE="br-int"
+OVS_VSWITCHD="ovs-vswitchd.service"
+ANTREA_DOCKER_IMAGE="projects.registry.vmware.com/antrea/antrea-ubuntu:v1.9.0"
```

Comment:
This will change for each version of antrea release.
So take version as an argument.